docs/manuals/source/DeveloperGuide/BuildAndTestBareos/systemtests.rst

```diff
@@ -71,6 +71,31 @@ sloppiness = file_macro
   hash_dir = false
   sloppiness = file_macro
 
+Prepare the PostgreSQL database for systemtests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Fedora 33 example
+^^^^^^^^^^^^^^^^^
+
+This is an example how to install the PostgreSQL server to use for systemtests.
+
+.. code-block:: shell-session
+  :caption: Install and configure postgreSQL Server for systemtests
+
+  sudo dnf install postgresql-server
+  /usr/bin/postgresql-setup --initdb
+  systemctl start postgresql
+  su postgres
+  createuser regress -s
+  psql --command='CREATE DATABASE regress OWNER regress TEMPLATE template0;'
+
+  # insert authentication methods for database user 'regress'
+  # BEFORE the general rules into file /var/lib/pgsql/data/pg_hba.conf
+  #
+  # local   all             regress                                 trust
+  # host    all             regress         127.0.0.1/32            trust
+  # host    all             regress         ::1/128                 trust
+
 Systemtests
```

Comment:
I would remove everything that is OS specific and only describe how to configure the database.

So "as use postgres do: createuser regress -s, psql, etc"
Then the title can be "How to configure postgresql for systemtests"

This is true for every OS that can run postgresql.